src/test/java/build/buildfarm/common/StringUtilsTest.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 The Bazel Authors. All rights reserved.
```

Comment:
Won't need an extra test if using an existing library :)

Reply:
thank you! way smarter!